Mathlib/Algebra/Order/SMul.lean

```diff
@@ -99,6 +124,9 @@ variable [OrderedSemiring R] [OrderedAddCommMonoid M] [SMulWithZero R M] [Ordere
     · exact (smul_lt_smul_of_pos hab hc).le
 #align smul_le_smul_of_nonneg smul_le_smul_of_nonneg
 
+-- TODO: Remove `smul_le_smul_of_nonneg` completely
```

Comment:
Can you elaborate on this? are you just trying to avoid the PR growing due to another rename, or is there a deeper issue?

Reply:
Yes, I am just trying to keep this PR contained.